### DISCUSSION TITLE
Welcome to ddddocr Discussions!

### DISCUSSION THREAD

**Discussion by sml2h3** [sml2h3/ddddocr#78] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Boris-code** [comment]
牛逼就完了

**Comment by tmjtp85** [comment]
ERROR: Ignored the following versions that require a different python version: 1.0.8 Requires-Python <3.8; 1.1.0 Requires-Python <3.9; 1.2.0 Requires-Python <3.10; 1.3.0 Requires-Python <3.10; 1.3.1 Requires-Python <3.10; 1.4.0 Requires-Python <3.10; 1.4.1 Requires-Python <3.10; 1.4.2 Requires-Python <3.10; 1.4.3 Requires-Python <3.10
ERROR: Could not find a version that satisfies the requirement onnxruntime (from ddddocr) (from versions: none)
ERROR: No matching distribution found for onnxruntime

python 版本是3.10.5
这是咋回事儿呢。

**Reply by doyoung1991** [reply]
我一直说哈希值不对，网上说是我的版本太高了，我是3.9的。

**Comment by byjames** [comment]
but 如果是 加、减、问号？（+ - ？   ）会识别错误 ？会识别2 或者7  + - 要么是不识别 识别除了数字也是错误的有时候会0

**Reply by JeffKko** [reply]
我只用英文辨識度蠻高的

**Comment by wang-xiaowu** [comment]
作者属实厉害,  而且针对png校验的正确率很高
